src/Controllers/ChatterDiscussionController.php

```diff
@@ -39,7 +37,7 @@ public function index(Request $request)
      */
     public function create()
     {
-        $categories = Category::all();
+        $categories = Models::category()->all();
```

Comment:
This will work. But you're right `all` is a static function. `All` should be `get`